modules/apps/asset/asset-browser-web/src/main/java/com/liferay/asset/browser/web/internal/display/context/AssetBrowserDisplayContext.java

```diff
@@ -422,6 +423,15 @@ public boolean isShowAssetEntryStatus() {
 	}
 
 	public boolean isShowBreadcrumb() {
+		Map<String, String[]> parameters =
+			_httpServletRequest.getParameterMap();
+
+		if (parameters.containsKey("scopeGroupType") &&
+			parameters.containsKey("scopeGroupClassPK")) {
+
+			return false;
+		}
+
 		ThemeDisplay themeDisplay =
```

Comment:
Step 3: If the scope's type and classPK were specified as attributes in the request, hide the breadcrumbs. Otherwise, show them.